src/client/components/pages/Home/index.tsx

```diff
@@ -6,11 +7,13 @@ const Home = () => (
     <div className="home__overview home__section">
       <div className="home__overview-container">
         <div className="home__overview-text">Your professional portfolio whenever you need it</div>
+        <div className="home__section home__features">
+          <SecondSection />
+        </div>
         <div className="home__section home__features">
           <Slider />
         </div>
-        <div className="home__section home__features">commin</div>
-        <div className="home__section home__features">soon</div>
+            <div className="home__section home__features">soon</div>
```

Comment:
the three sections have the same 2 classes. I'd say the lastone `home__features` is incorrect or not necessary